plugin/trino-hive/src/main/java/io/trino/plugin/hive/fs/DirectoryLister.java

```diff
@@ -27,4 +27,7 @@
 {
     RemoteIterator<LocatedFileStatus> list(FileSystem fs, Table table, Path path)
```

Comment:
As far as I understand, both `list` operations are listing **ONLY** files (and not sub-directories).

Please take into consideration changing the method name. e.g. : `listRecursively`